app/controllers/providers/delete_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Providers
```

Comment:
I've just spotted `Backable`. On first glance it didn't seem like there was anything in there I could use, but feel free to correct me if I'm wrong. I'm also happy to put this functionality in there if it would be helpful

Reply:
I'd say maybe put a todo here and we can look after a while?